public/app/features/datasources/NewDataSourcePage.tsx

```diff
@@ -131,37 +139,90 @@ const DataSourceTypeCard: FC<DataSourceTypeCardProps> = (props) => {
   // find first plugin info link
   const learnMoreLink = plugin.info?.links?.length > 0 ? plugin.info.links[0] : null;
 
+  const styles = useStyles2(getStyles);
+
   return (
-    <Card
-      title={plugin.name}
-      description={plugin.info.description}
-      ariaLabel={selectors.pages.AddDataSource.dataSourcePlugins(plugin.name)}
-      logoUrl={plugin.info.logos.small}
-      actions={
-        <>
-          {learnMoreLink && (
-            <LinkButton
-              variant="secondary"
-              href={`${learnMoreLink.url}?utm_source=grafana_add_ds`}
-              target="_blank"
-              rel="noopener"
-              onClick={onLearnMoreClick}
-              icon="external-link-alt"
-            >
-              {learnMoreLink.name}
-            </LinkButton>
-          )}
-          {!isPhantom && <Button disabled={plugin.unlicensed}>Select</Button>}
-        </>
-      }
-      labels={!isPhantom && <PluginSignatureBadge status={plugin.signature} />}
-      className={isPhantom ? 'add-data-source-item--phantom' : ''}
-      onClick={onClick}
-      aria-label={selectors.pages.AddDataSource.dataSourcePlugins(plugin.name)}
-    />
+    <Card className={cx(styles.card, 'card-parent', { 'add-data-source-item--phantom': isPhantom })} onClick={onClick}>
+      <Card.Heading
+        className={styles.heading}
+        aria-label={selectors.pages.AddDataSource.dataSourcePluginsV2(plugin.name)}
+      >
+        {plugin.name}
+      </Card.Heading>
+      <Card.Figure align="center" className={styles.figure}>
+        <img className={styles.logo} src={plugin.info.logos.small} alt="" />
+      </Card.Figure>
+      <Card.Description className={styles.description}>{plugin.info.description}</Card.Description>
+      {!isPhantom && (
+        <Card.Meta className={styles.meta}>
+          <PluginSignatureBadge status={plugin.signature} />
+        </Card.Meta>
+      )}
+      <Card.Actions className={styles.actions}>
+        {learnMoreLink && (
+          <LinkButton
+            variant="secondary"
+            href={`${learnMoreLink.url}?utm_source=grafana_add_ds`}
+            target="_blank"
+            rel="noopener"
+            onClick={onLearnMoreClick}
+            icon="external-link-alt"
+            aria-label={`${plugin.name}, learn more.`}
+          >
+            {learnMoreLink.name}
+          </LinkButton>
+        )}
+      </Card.Actions>
+    </Card>
   );
 };
 
+function getStyles(theme: GrafanaTheme2) {
+  return {
+    heading: css({
+      fontSize: theme.v1.typography.heading.h5,
+      fontWeight: 'inherit',
+    }),
+    figure: css({
+      width: 'inherit',
+      marginRight: '0px',
+    }),
+    meta: css({
+      marginTop: '6px',
```

Comment:
can we get this from the theme? or does this specifically have to be 6px exactly? 🤔